lib/ancestry/has_ancestry.rb

```diff
@@ -30,6 +30,8 @@ def has_ancestry options = {}
       validates_format_of self.ancestry_column, :with => derive_ancestry_pattern(options[:primary_key_format]), :allow_nil => true
       extend Ancestry::MaterializedPath
 
+      include Ancestry::MaterializedPathPg if ActiveRecord::Base.connection.adapter_name.downcase == "postgresql"
```

Comment:
introduce `update_strategy` that will allow us to opt into this behavior

please keep the postgresql check here
but add a check to make sure update strategy is sql



for those who want this everywhere, add a module level variable with default update strategy